.travis.yml

```diff
@@ -40,7 +39,7 @@ matrix:
       env:
         - XCODE_SDK=watchsimulator
         - XCODE_ACTION=build
-        - XCODE_DESTINATION="platform=watchOS Simulator,name=Apple Watch - 38mm"
+        - XCODE_DESTINATION="platform=watchOS Simulator,name=Apple Watch - 38mm,OS=2.2"
```

Comment:
It's surprising that there are duplicated simulators of watchOS 3.2 in this time 😲